src/app/calculator/furnaces/flue-gas/flue-gas.service.ts

```diff
@@ -30,19 +33,17 @@ export class FlueGasService {
   }
 
   calculate(settings: Settings) {
+    this.initDefaultEmptyOutput();
+    let output = this.output.getValue();
```

Comment:
whole function has a bunch of temporary variables without types. add types please